CMakeLists.txt

```diff
@@ -95,6 +77,9 @@ add_executable(${XTENSOR_BENCHMARK_TARGET} ${XTENSOR_BENCHMARK} ${XTENSOR_HEADER
 
 # Mandatory dependencies
 # ======================
+set(CMAKE_LIBRARY_PATH "$ENV{CONDA_PREFIX}/lib")
+set(CMAKE_INCLUDE_PATH "$ENV{CONDA_PREFIX}/include")
```

Comment:
This will prevent building the project outside of a conda environment. The recommended way to set these variables is to set the CMAKE_INSTALL_PREFIX variable when invoking cmake:

```
cmake -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX
```

The folder tree is a bit different on Windows:

```
cmake -DCMAKE_INSTALL_PREFIX=%CONDA_PREFIX%/Library
```

Reply:
oh, see above. I'll remove this. 

> The recommended way to set these variables is to set the CMAKE_INSTALL_PREFIX variable when invoking cmake 

I am building with that command. 
Bad thing is however, some libraries are still not detected 
https://github.com/xtensor-stack/xtensor-benchmark/issues/13#issuecomment-639183702

Reply:
Can you try the following and see if this fixes the pythran error?

```
cmake -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DPYTHON_EXECUTABLE=`which python`
```

Reply:
no, python was detected correctly before, now the variable is unused
```
Manually-specified variables were not used by the project:

PYTHON_EXECUTABLE
```

I think the bigger problem is why numpy isn't picked up?

Reply:
Looks like a typo [here](https://github.com/xtensor-stack/xtensor-benchmark/blob/master/CMakeLists.txt#L138), it should be `find_package(Numpy)` (lower p) so that the name mathes that of the .cmake file in `modules`

Reply:
`Numpy` is resolved :+1: let's take on others